ui/gizmos.js

```
@@ -97,11 +97,11 @@ function registerBindings() {
     deleteBlockWithUndo(meshBlockIdMap[blockKey]);
   });
   // Exit gizmo with Tab key
-  InputManager.on("GIZMO", "Tab", (e) => {
+  KeyboardDispatcher.on("GIZMO", "Tab", (e) => {
```

ESLint

[error] 'e' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)